src/main.rs

```diff
@@ -47,9 +52,16 @@ fn load_filetypes_database(matches: &ArgMatches, user_config_path: &PathBuf) ->
 
     let database_path = database_path_from_arg
         .or(database_path_env)
-        .or_else(|| util::get_first_existing_path(&[&database_path_user, database_path_system]))
-        .ok_or(VividError::CouldNotFindDatabase)?;
```

Comment:
Can we remove `VividError::CouldNotFindDatabase` in that case?
```
warning: variant is never constructed: `CouldNotFindDatabase`
  --> src/error.rs:15:5
   |
15 |     CouldNotFindDatabase,
   |     ^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default

```

Reply:
I think so, although we will need to keep the CouldNotFindTheme error